app/API/v1/views/political_office.py

```diff
@@ -4,42 +4,48 @@
 from app import app, api
 
 political_office = reqparse.RequestParser()
-political_office.add_argument('office_type', type=str, help='Please enter your office_type', required=True)
+political_office.add_argument('officetype', type=str, help='Please enter your officetype', required=True)
 political_office.add_argument('name', type=str, help='Please enter your name', required=True)
 
 class Office(Resource):
   def post(self):
     arguments = political_office.parse_args()
-    office_type = arguments['office_type']
+    officetype = arguments['officetype']
     name = arguments['name']
 
-    if not office_type or not name:
-      return{"message": "Provide office and name"}, 400
+    if not officetype:
+      return{"status":400, "message": "Provide office name"}, 400
 
-    if not office_type.isalpha():
-      return{"message":"Office type should be alphabet"}, 400
+    if not name:
+      return{"status":400, "message": "Provide name"}, 400
+
+    if not officetype.isalpha():
+      return{"status": 400, "message":"Office type should be alphabet"}, 400
+
+    if not name.isalpha():
+      return {"status":400, "message": "Name type should be alphabet"}, 400
 
     for office in offices:
-      if office_type in office.values() or name in office.values():
-        return {"message": "either office or name already exists"}, 400
+      if officetype in office.values() or name in office.values():
+        return {"status": 409, "message": "either office or name already exists"}, 409
 
     office = OfficesModel()
-    office = office.save(office_type, name)
-    return {"message": "Office created successfully" ,"data": [office] }, 201
+    office = office.save(officetype, name)
+    return {"status":201, "data": [office] }, 201
 
 class SingleOffice(Resource):
   def get(self,id):
     single_office = OfficesModel()
     if not single_office.single_office(id):
-      return {"message": "page not availabe"}, 400
-    return single_office.single_office(id), 201
+      return {"status":404," message": "page not availabe"}, 404
+    return {"status":201, "data": single_office.single_office(id)}, 201
 
 class GetAllOffice(Resource):
   def get(self):
     office = OfficesModel()
     if not office.all_office():
-      return {"message": "page not availabe"}, 400
-    return office.all_office()
+      return {"status":404, "message": "page not availabe"}, 404
```

Comment:
What does `page not available` mean to a user? Consider using more descriptive terms